Front end penúltimo commit

ArTezra1 opened the pull request:
sei lá mudei uns bag ai e tem q subir a api no ar e mudar todas as variáveis de rota da api pra funcionar e não precisar baixar as dependências locais.

EstevaM-prog committed:
Merge pull request #15 from EstevaM-prog/branch-Lipe

att

ArTezra1 committed:
Penúltima commit

EstevaM-prog reviewed (approved):
Mais tarde, vou revisar o cod e tentar solucionar os conflitos

LipeOliver-prog committed:
arrumei as img na barra de pesquisa

Arrumei as img na barra de pesquisa, ajustando os arrays e sua localização de chamada.

LipeOliver-prog committed:
exclui oagina pesquisa

exclui pagina de pesquisa

EstevaM-prog committed:
pequenas alterações
